src/bin/rustup-init.rs

```diff
@@ -33,24 +32,19 @@ use rustup::errors::RustupError;
 use rustup::is_proxyable_tools;
 use rustup::utils::utils::{self, ExitCode};
 
-fn main() {
+#[tokio::main]
+async fn main() {
     #[cfg(windows)]
     pre_rustup_main_init();
 
     let process = Process::os();
-    Builder::new_multi_thread()
-        .enable_all()
-        .build()
-        .unwrap()
-        .block_on(async move {
-            match maybe_trace_rustup(&process).await {
-                Err(e) => {
-                    common::report_error(&e, &process);
-                    std::process::exit(1);
-                }
-                Ok(utils::ExitCode(c)) => std::process::exit(c),
-            }
-        });
+    match maybe_trace_rustup(&process).await {
```

Comment:
Just found out that we're already tracing 100% of the time so the function name makes no sense now. Given that it's yet another single-use function, I'm inclined to inlining it here :)

Reply:
Done.